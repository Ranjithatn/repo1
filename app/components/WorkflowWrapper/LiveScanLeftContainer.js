import React, { Fragment } from "react";

function LeftContainer({ formatMessage, heading, content, annotation,mugshot }) {
  return (
    <Fragment>
      <div className="header">{heading}</div>
      <div className="dark-container">{content}</div>    
      <div className="mugshot">{mugshot}</div>
      <div className="annotations">{annotation}</div>
      {/*<div className="mugshot">{mugshot}</div>*/}
    </Fragment>
  );
}

export default LeftContainer;
