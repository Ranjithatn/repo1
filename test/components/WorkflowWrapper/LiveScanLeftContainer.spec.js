import React from "react";
import toJson from "enzyme-to-json";
import LiveScanLeftContainer from "../../../app/components/WorkflowWrapper/LiveScanLeftContainer";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });


describe('<LiveScanLeftContainer />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <LiveScanLeftContainer  />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });

});
