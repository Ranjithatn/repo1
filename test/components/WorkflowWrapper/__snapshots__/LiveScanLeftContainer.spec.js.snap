// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LiveScanLeftContainer /> should render without crashing 1`] = `
<LeftContainer>
  <div
    className="header"
  />
  <div
    className="dark-container"
  />
  <div
    className="mugshot"
  />
  <div
    className="annotations"
  />
</LeftContainer>
`;
